src/patternfly/components/Accordion/examples/Accordion.md

```diff
@@ -294,6 +295,62 @@ cssPrefix: pf-v5-c-accordion
 {{/accordion}}
 ```
 
+### Toggle icon at start
+
+```hbs
```

Comment:
Not sure if we need it, but you could add `isBeta` so we can iterate on it for a bit if we need to make any changes or anticipate it being a beta prop in react. I probably wouldn't since it's so simple, but totally your call.